test/interface/static_array_tests.jl

```diff
@@ -2,36 +2,43 @@ using StaticArrays, Test
 using OrdinaryDiffEq
 using RecursiveArrayTools
 
-u0 = fill(zero(MVector{2,Float64}), 2)
-u0[1] = ones(MVector{2,Float64}) .+ 1
+u0 = [fill(2, MVector{2,Float64}), ones(MVector{2,Float64})]
 f = (du,u,p,t) -> du .= u
 ode = ODEProblem(f, u0, (0.,1.))
 sol = solve(ode, Euler(), dt=1e-2)
+@test !any(iszero.(sol(1.0))) && !any(sol(1.0) .== u0)
```

Comment:
Just a minor suggestion (feel free to ignore it):
```suggestion
@test !any(iszero, sol(1.0)) && !any(sol(1.0) .== u0)
```